packages/headless/src/controllers/result-list/headless-result-list.ts

```diff
@@ -68,14 +93,6 @@ export function buildResultList(
     /**
      * Using the same parameters as the last successful query, fetch another batch of results. Particularly useful for infinite scrolling, for example.
      */
-    fetchMoreResults() {
-      if (this.state.isLoading) {
-        engine.logger.warn(
```

Comment:
Remove the warning as, for a infinite scrolling implementation, it can be normal to call fetchMoreResults many times successively